backend/main_test.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+func mustTime(s string) time.Time {
+	t, err := time.Parse("2006-01-02", s)
+	if err != nil {
+		panic(err)
+	}
+	return t
+}
+
+func TestParse(t *testing.T) {
+	s := `
+Date, 	    Coutry, Province, Lat, Long, Confirmed, Recovered, Deaths
+2020-01-01,    Abc,         ,   2,    1,         3,         0,      1
+2020-01-02,    Abc,         ,   2,    1,        10,         2,      6
+2020-01-03,    Abc,         ,   2,    1,        12,         4,      5
+`
+	b := bytes.NewBufferString(s[1:])
```

Comment:
I wonder if `s[1:]` works on windows which uses CR,LF as line terminators? Perhaps trim it or find the first non-whitespace? I wonder if you even care about windows?

Reply:
you are right-  i don't care about windof (as we lovingly call it in german -  doof means dumb).
changed to TrimSpace